src/App.jsx

```diff
@@ -23,23 +23,42 @@ export const alex = {
   isMarried: false,
 };
 
-export const App = () => (
-  <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
+function Person({ name, age, sex, isMarried, partnerName }) {
+  let partner = '';
 
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">Maksym is my husband</p>
-    </section>
+  if (sex === 'm') {
+    partner = 'wife';
+  } else {
+    partner = 'husband';
+  }
 
+  return (
     <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      <p className="Person__age">
+        {!age ? (
+          ''
+        ) : (
+          `I am ${age}`
+        )
+        }
+      </p>
```

Comment:
Handle truth values first
```suggestion
      {age && 
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      }
```